### ISSUE TITLE
Create a new body for any change(sticky-version)

### ISSUE THREAD

**Issue by av-dev** [GuillaumeJasmin/react-table-hoc-fixed-columns#10]
This line leads to the creation of a new body when calling the render function
```
TbodyComponent={propsTBody =>(
    <div><DefaultReactTable.defaultProps.TbodyComponent {...propsTBody} /></div>
)}

```

**Comment by GuillaumeJasmin** [comment]
Thanks ! I had to wrap TBody into `<div>` to keep Footer position working, because `overflow: visible !important` on tbody break the Footer position.

I will fixed like this:
```js
TbodyComponent={propsTBody => (
    <div>
      {props.TbodyComponent
        ? props.TbodyComponent(propsTBody)
        : <DefaultReactTable.defaultProps.TbodyComponent {...propsTBody} />
      }
    </div>
)}
```

**Comment by av-dev** [comment]
@GuillaumeJasmin This will not solve the problem of inserting a new Tbody in the DOM with each re-renderer

**Comment by GuillaumeJasmin** [comment]
What do you suggest ?

**Comment by GuillaumeJasmin** [comment]
The bug appear with `v1.0.0-beta.8`, because of `overflow: visible !important` on `.rt-tbody`

[See bug on codesandbox](https://codesandbox.io/s/p57nw9p8v0)

**Comment by GuillaumeJasmin** [comment]
I made a mistake, I mixed up 2 bugs, but that not fix the creation of new body. My solution fix another issue, sorry.

**Comment by av-dev** [comment]
@GuillaumeJasmin Set `flex: 99999 0 auto` for `.rt-tbody` and remove `TbodyComponent={...}` in the HOC. This will solve both problems that we mentioned in this issue

**Comment by GuillaumeJasmin** [comment]
This dosen't work, the footer is place after the last row, and it's not always visible, so main feature of Footer is broken. Footer must be always visible, juste above the pagination area.

**Comment by GuillaumeJasmin** [comment]
Great, it seems to works !
`.rt-body` also work with `flex-grow` simply set to `1` instead of `99999` .
I will update it and remove `TbodyComponent`

Thanks !

**Comment by GuillaumeJasmin** [comment]
fix in `v1.0.0-beta.12`
